examples/ansible3/validators/check_interfaces.py

```diff
@@ -0,0 +1,13 @@
+"""Example validator plugin."""
+from schema_enforcer.schemas.validator import JmesPathModelValidation
+
+
+class CheckInterface(JmesPathModelValidation):  # pylint: disable=too-few-public-methods
+    """Check that each device has more than one core uplink."""
+
```

Comment:
I think pydocstyle requires no newline here?

Reply:
No newline after the comment? I vaguely remember that pydocstyle complained when it isn't in a single line or you miss the period at the end. This validator is also in the test directory and pydocstyle is passing.

Reply:
Ack. That's interesting -- I've seen pydocstyle flag when a function has space between the docstring and the beginning of the logic executed by the function. It may just be on multiline docstrings though.. IDK. If pydocstyle isn't complaining I'm good :).